.github/workflows/aws.yml

```
@@ -0,0 +1,47 @@
+
+on:
+  push:
+    branches:
+     # - development
+      - testpipeline
+  pull_request:
+    branches:
+      - testpipeline
+
+name: Sslscan Workflow
+
+jobs:
+  check-format:
+    name: Check format
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Check sources format
+        run: crystal tool format --check
+
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1      
+      - name: Unit testing
+        run: shards install && crystal spec
+  ameba:
```

Let's add a newline here to be consistent

```suggestion

    ameba:
```